indico/MaKaC/accessControl.py

```diff
@@ -60,6 +60,7 @@ def __init__( self, owner ):
         self.contactInfo = ""
 
     def getOwner(self):
+        raise NotImplementedError('getOwner')
         return self.owner
```

Comment:
No, I just didn't want to start removing the code now. And in some other cases I kept it to make it easier to look at the previous logic if needed (eventually all those methods go away completely)